### ISSUE TITLE
Issues using kubectl

### ISSUE THREAD

**Issue by chiradeep** [skippbox/boot2k8s#5]
- `kubectl` commands often time out. E.g., kubectl create -f frontend-controller.yaml 
  error: could not read an encoded object from frontend-controller.yaml: unable to connect to a server to handle "replicationcontrollers": couldn't read version from server: Get http://localhost:8080/api: EOF
- Works more consistently if I give a second 'host-only' network to the boot2k8s VM (configure VBox DHCP to give it 192.168.64.100) and use that IP as the target for kubectl
- Downloading of images is flaky. I wonder if giving the boot2k8s a proper disk (like boot2docker /dev/sda1) will help. 
- As of now, it is quite unusable

**Comment by sebgoa** [comment]
@chiradeep yes, I have seen this erratic behavior. It works better if you start with our fork of docker-machine. https://github.com/skippbox/machine which indeed ads a second host only interface.

To make it work standalone, I need to check the Vbox commands to add that interface and add a persistent disk. I have seen issues when the VM pauses and the docker images disappear...

**Comment by chiradeep** [comment]
Yes, I was wondering if docker-machine vbox + docker-compose could achieve the same effect.

**Comment by sebgoa** [comment]
yes it can. but then it's two commands plus a config file.
